client/py/synnax/channel/retrieve.py

```diff
@@ -162,6 +189,14 @@ def retrieve(self, params: ChannelParams) -> list[ChannelPayload]:
         results.extend(retrieved)
         return results
 
+    def retrieve_one(self, param: ChannelKey | ChannelName) -> ChannelPayload | None:
```

Comment:
Add test

Reply:
This is a function that is only used internally, so it's not publicly exposed for testing. It gets hammered pretty hard by publicly available test cases.

Reply:
Actually went in and added cases